docs/userGuide/syntax/pageLayouts.mbdf

```diff
@@ -114,27 +114,18 @@ which injects the actual page content in every page. This allows you to build la
 
 {{ icon_example }} Adding statistics formula to the bottom of every page
 
-<box>
-<i>index.md</i>
-<br><br>
-
-  ```html
+  ```html {heading="index.md"}
   #### Main content of your page
 
   Content of your page
   ```
-</box>
 
-<box>
-<i>page.md</i>
-<br><br>
-  {{ mainContentBody }}
-  ```html
+  ```html {heading="page.md", highlight-lines="1"}
+     MAIN_CONTENT_BODY
```

Comment:
I think this should be a variable. (previously this was `{{ mainContentBody }}`)

Reply:
When I am putting is a variable this is how it gets rendered

![Screen Shot 2020-03-12 at 6 02 15 PM](https://user-images.githubusercontent.com/35946746/76509832-98805380-648b-11ea-8007-4280e685c2c8.png)

How do you want me to implement it?

Reply:
Seems like we would need to be able to escape the variable in the code block in order to render it as is rather than replace it with the actual value.

There is a PR that aims to address this (https://github.com/MarkBind/markbind/pull/1049) but I haven't had the time to review that yet. 

Alternatively, we could do a _dirty hack_ and use the zero width space to break up the variable syntax when declaring the `mainContentBody` variable:
```
<variable name="mainContentBody">
{&#8203;{ MAIN_CONTENT_BODY }}
</variable>
```
If we do this, we should leave a todo reminding us to change it to use the escape tags once that PR is ready. 

I'd lean towards the latter, so we can make a clean move towards code blocks and avoid any potential merge conflicts if there are any new changes to the documentation, since I'm not sure when that PR is going to be merged.

Reply:
Okay will do accordingly.